File: client/src/main/java/com/virjar/model/Phone.java
package com.virjar.model;

import java.io.Serializable;

/**
 * Description: Phone
 *
 * @author lingtong.fu
 * @version 2016-09-05 00:01
 */
public class Phone implements Serializable {

    private static final long serialVersionUID = -6732131032651082674L;

    private String province;
    private String city;
    private short mmo;

    public Phone() {
    }

    public Phone(String province, String city, short mmo) {
        this.province = province;
        this.city = city;
        this.mmo = mmo;
    }

    public String getProvince() {
        return province;
    }

    public void setProvince(String province) {
        this.province = province;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public short getMmo() {
        return mmo;
    }

    public void setMmo(short mmo) {
        this.mmo = mmo;
    }

    @Override
    public String toString() {
        return "Phone [" +
                "province=" + province +
                ", city=" + city +
                ", mmo=" + mmo +
                "]";
    }
}

File: proxyipcenter-dotnet/proxyipcenter-dotnet/Model/Phone.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proxyipcenter_dotnet.Model
{
    [Serializable]
    public class Phone
    {
        private string province;
        private string city;
        private short mmo;

        public string Province
        {
            get
            {
                return province;
            }

            set
            {
                province = value;
            }
        }

        public string City
        {
            get
            {
                return city;
            }

            set
            {
                city = value;
            }
        }

        public short Mmo
        {
            get
            {
                return mmo;
            }

            set
            {
                mmo = value;
            }
        }

        public Phone()
        {

        }

        public Phone(string p,string c,short m)
        {
            province = p;
            city = c;
            mmo = m;
        }

        public override string ToString()
        {
            object[] args = { province, city, mmo };
            return string.Format("Phone [province={0}, city={1}, mmo={2}", args);
        }
    }
}
